rootfs/etc/services.d/pimd/finish

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/execlineb -S0
+# ==============================================================================
+# Take down the S6 supervision tree when pimd fails
+# ==============================================================================
+if { s6-test ${1} -ne 0 }
+if { s6-test ${1} -ne 256 }
+
+s6-svscanctl -t /var/run/s6/services
```

Comment:
I think we not need a finish for now but if you like, use the new s6-overlay commands: https://github.com/home-assistant/addons-example/blob/main/example/rootfs/etc/services.d/example/finish